### PR TITLE
[Archives-ouvertes.fr] Simplify and update ruleset

### PR DESCRIPTION
**Pull request by Bisaloo**

### CODE REVIEW

File: src/chrome/content/rules/Archives-ouvertes.fr.xml [hunk 1]
```diff
@@ -1,48 +1,34 @@
...
 
 	<target host="archives-ouvertes.fr" />
 	<target host="*.archives-ouvertes.fr" />
-
-
-	<!--	Not secured by server:
-					-->
-	<!--securecookie host="^tel\.archives-ouvertes\.fr$" name="^PHPSESSID$" /-->
+		<test url="http://www.archives-ouvertes.fr/" />
+		<test url="http://archives-ouvertes.fr/" />
```
Comment (J0WI) [inline src/chrome/content/rules/Archives-ouvertes.fr.xml]: test url already covered by target

File: src/chrome/content/rules/Archives-ouvertes.fr.xml [hunk 2]
```diff
@@ -1,48 +1,34 @@
...
+		to="https://archives-ouvertes.fr/" />
 
-	<!--	Redirect keeps path and args:
-						-->
-	<rule from="^http://(?:www\.)?archives-ouvertes\.fr/+"
-		to="https://hal.archives-ouvertes.fr/" />
-
-	<rule from="^http://(api|aurehal|aurehal-preprod|hal|hal-\w+|halshs|pastel|tel)\.archives-ouvertes\.fr/"
+	<rule from="^http://(api|aurehal|aurehal-preprod|cel|cv|doc|hal|hal-\w+|halshs|medihal|pastel|tel|telearn|www)\.archives-ouvertes\.fr/"
 		to="https://$1.archives-ouvertes.fr/" />
```
Comment (J0WI) [inline src/chrome/content/rules/Archives-ouvertes.fr.xml]: Why aren't you using a generic http to https rule here?

Comment (Bisaloo) [reply src/chrome/content/rules/Archives-ouvertes.fr.xml]: I'm not sure. I simply kept the old rule. I guess it makes sense if some targets don't support `https`.

File: src/chrome/content/rules/Archives-ouvertes.fr.xml [hunk 3]
```diff
@@ -1,48 +1,34 @@
...
-		- aurehal
-		- aurehal-preprod
-		- hal
-		- hal-\w+ *
-		- halshs
-		- medihal
-		- pastel
-		- tel
-
-	* Per-institution domains
```
Comment (J0WI) [inline src/chrome/content/rules/Archives-ouvertes.fr.xml]: We should keep a comment about the wildcard.

**Pull request merged**